pkgs/development/python-modules/yowsup/default.nix

```diff
@@ -3,9 +3,8 @@
 }:
 
 buildPythonPackage rec {
-  name = "${pname}-${version}";
   pname = "yowsup";
-  version = "2.5.2";
+  version = "2.5.7";
 
   # python2 is currently incompatible with yowsup:
   # https://github.com/tgalal/yowsup/issues/2325#issuecomment-343516519
```

Comment:
Could you please update this message so it mentions the six==1.11 incompatibility?